mephisto/abstractions/providers/mock/wrap_crowd_source.js

```diff
@@ -15,6 +15,11 @@ Returning None for the assignment_id means that the task is being
 previewed by the given worker.
 \------------------------------------------*/
 
+// The following is the nanoevents npm library (link) manually processed as such:
+// 1. transpiled to support more browser targets using the Babel transpiler (link), and
+// 2. minified using a JS minifier (link)
```

Comment:
`(link)` from my comment meant to actually replace that with the link (you can grab the links from my comment as well.

The babel one you might want to clean up a bit and remove the params otherwise it's really long and also includes the source code in the url